src/Framework/Profiler/EvaluationLocation.cs

```diff
@@ -77,47 +93,56 @@ public struct EvaluationLocation
         public string ElementName { get; }
 
         /// <nodoc/>
-        public string ElementOrCondition { get; }
+        public string Description { get; }
```

Comment:
What's the difference between Description and EvaluationDescription?

Reply:
Renamed EvaluationDescription to EvaluationPassDescription, and Description to ElementDescription

---
In reply to: [156443221](https://github.com/Microsoft/msbuild/pull/2799#discussion_r156443221) [](ancestors = 156443221)